src/container/internal.tsx

```diff
@@ -100,6 +103,10 @@ export default function InternalContainer({
   // In this case we don't want the container to have sticky styles, as only the table header row will show as stuck on scroll.
   const shouldHaveStickyStyles = isSticky && !isMobile;
 
+  function getMediaStyles() {
+    return mediaPosition === 'top' ? { height: mediaHeight } : { width: mediaWidth };
```

Comment:
Why does this need to be a function? We can just say 

```
const mediaStyles = mediaPosition === 'top' ? { height: mediaHeight } : { width: mediaWidth };
```

Reply:
Leftover from refactoring, will remove this!

Reply:
Done!